cloud-build/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# This Dockerfile spins up a container where presubmit tests are run.
+# Cloud Build orchestrates this process.
+
+FROM gcr.io/cloud-builders/gcloud
+
+# Copy everything into the container
+COPY . init-actions/
```

Comment:
Cloud Build mounts GitHub repo to `/workspace` directory (current directory here). You need to copy it to new directory only if you are running multiple containers in the same build, otherwise you can just run presubmit in the workspace.

You can safely delete `COPY` and `WORKDIR` commands